WorkloadTools/Consumer/WorkloadFile/WorkloadFileWriterConsumer.cs

```diff
@@ -70,6 +71,19 @@ public class WorkloadFileWriterConsumer : BufferedWorkloadConsumer
                     $writes
                 );";
 
+        private string update_events = @"
+                UPDATE Events SET cpu = $cpu,
```

Comment:
Two major concerns:
1. I am not sure that this will work 100% of the time. This code updates the first event (for this session) without a duration, but how can we tell it is the correct one?
2. Performance: the previous version could deal with very intensive workloads because it was just inserting data. This update introduces much more load. Did you test a very intensive workload?